Lesson 03/news_reader/index.js

```diff
@@ -0,0 +1,47 @@
+const request = require('request');
+const clc = require('cli-color');
+const cli = require('cli');
+const cliOptions = {
+	newsCount: ['n', 'Fetch first NUM of news.', 'num']
+};
+
+cli.setApp('A simple news reader app.', '0.0.1');
+cli.enable('version');
+cli.parse(cliOptions);
+cli.options.newsCount = cli.options.newsCount || 10;
+	
+request('https://www.vesti.ru/news/', (err, res, body) => {
+	if (err) {
+		console.error(err);
+		
+		return;
+	}
+	
+	if (res.statusCode !== 200) {
+		console.log('Request status code: ' + res.statusCode);
+		
+		return;
+	}
+	
+	console.log('First ' + cli.options.newsCount + ' news from ' +
+					clc.bold.yellowBright('www.vesti.ru'));
+	
+	const $ = require('cheerio').load(body);
+	const $newsItems = $('.b-item__inner', '.news-wrapper_');
+	const newsCountToFetch = 10;
+	let $newsInnerItem, $newsItem, newsTitle, newsText, newsTime;
+	
+	for (let i = 0; (($newsItem = $newsItems[i]) && i < newsCountToFetch); i++) {
+		$newsInnerItem = $('.b-item__title a', $newsItem);
+		newsTitle = $newsInnerItem[0].children[0].nodeValue.trim();
```

Comment:
I'd recommend you using high-level methods of cheerio instead of children.

Reply:
Agreed, will use a method text() instead.